src/sql/workbench/contrib/notebook/browser/cellViews/linkHandler.directive.ts

```diff
@@ -71,6 +72,9 @@ export class LinkHandlerDirective {
 				return;
 			}
 		}
+		// On Mac, the link href scheme is vscode-file. We want to convert the vscode-file URI to file URI
```

Comment:
I'd just simplify this to 

`Convert vscode-file protocol URIs to file since that's what Notebooks expect to work with`

The mac vs Windows thing is more of an odd detail and doesn't really change anything here (since we want to be doing that regardless). And there might be some confusion if people see this and wonder why this isn't being done only on Macs.